src/kibana/components/vislib/components/zero_injection/flatten_data.js

```diff
@@ -18,9 +18,10 @@ define(function (require) {
         charts = obj.rows ? obj.rows : obj.columns;
       }
 
-      return _.chain(charts ? charts : [obj])
+      return _(charts ? charts : [obj])
       .pluck('series')
       .flatten()
+      .filter(Boolean)
       .pluck('values')
       .flatten()
```

Comment:
Lets move this filter to the bottom of the chain so that if a series object doesn't have values for some reason we don't have the same problem.